tools/sense_studio/templates/testing.html

```diff
@@ -0,0 +1,136 @@
+
+        <div class="uk-child-width-1-1@s uk-grid-small" uk-grid>
+            <div>
+                 <div class="uk-form-controls">
+                     <div class="uk-width-1-1">
+                        <label>
+                            <input class="uk-checkbox" type="checkbox" name="saveVideo" onclick="toggleOutputVideoField();">
+                            &nbsp;&nbsp;Save Output Video
+                        </label>
+                     </div>
+                </div>
+            </div>
+        </div>
+
+        <div id="inputVideoDiv" class="uk-child-width-1-1@s uk-grid-small uk-hidden" uk-grid>
+            <div>
+                <label class="uk-form-label" for="inputVideoPath">Input video path</label>
+                <label class="uk-form-label uk-text-danger" id="inputVideoPathLabel"></label>
+                <div class="uk-form-controls">
+                    <div class="uk-inline uk-width-1-1">
+                        <span class="uk-form-icon" uk-icon="icon: video-camera"></span>
+                        <input class="uk-input file-path-search" type="text" id="inputVideoPath" placeholder="/path/to/input-video"
+                               autocomplete="off" oninput="getPathInputs('inputVideoPathLabel', 'inputVideoPath');">
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div id="outputVideoDiv" class="uk-child-width-1-1@s uk-grid-small uk-hidden" uk-grid>
+             <div>
+                <label class="uk-form-label" for="outputVideoName">
+                    Output video name ({{ output_path_prefix }}..) <span class="uk-label">.mp4</span>
+                </label>
+                <div class="uk-form-controls">
+                    <div class="uk-inline uk-width-1-1">
+                        <span class="uk-form-icon" uk-icon="icon: video-camera"></span>
+                        <input class="uk-input" type="text" id="outputVideoName" placeholder="Output video name"
+                               autocomplete="off">
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <button class="uk-button uk-button-primary uk-margin uk-margin-right" id="btnTest" type="button" tabindex="0"
+                onclick="startTesting('{{ url_for('.start_testing') }}');"
+                {% if not classifiers %} disabled {% endif %}>
+            Test model
+        </button>
+        <button class="uk-button uk-button-danger" id="btnCancelTest" type="button" tabindex="1"
+                onclick="cancelTesting('{{ url_for('.cancel_testing') }}');" disabled>
+            Cancel
+        </button>
+    </form>
+
+    <!--    Output video streaming frame-wise (Automatically filled) -->
+    <div id="videoStream" class="uk-margin uk-text-center">
+        <h4> Output Video Stream</h4>
+        <div class="uk-section uk-section-muted uk-padding-remove-vertical uk-inline">
+            <img id="frame" src="" height="480" width="640">
```

Comment:
Can we remove the border around this box so that it looks similar to the one on the recording page?